assets/large_language_models/rag/components/src/flow_creation.py

```diff
@@ -166,6 +174,15 @@ def main(args, ws, current_run, activity_logger: Logger):
     print("embedding_deployment_name: %s" % embedding_deployment_name)
     print("embedding_model_name: %s" % embedding_model_name)
 
+    if completion_model_name.startswith("gpt-"):
```

Comment:
What else would this be?

Reply:
it used to be davinci-003, but that not used much anymore.

Reply:
in case someone was still using text-davinci I guess. I just parroted the "if the model is gpt-35-turbo or gpt-4 we want to use chat flows" from the ES logic Prakhar added